src/functions-and-arrays.js

```diff
@@ -1,73 +1,144 @@
 // Iteration #1: Find the maximum
 
+function maxOfTwoNumbers(num1, num2){
+  if(num1 > num2){
+    return num1;
+   } else {
+     return num2;
+   }
+};
+
 // Iteration #2: Find longest word
 const words = ['mystery', 'brother', 'aviator', 'crocodile', 'pearl', 'orchard', 'crackpot'];
 
+function findLongestWord(arr) {
+if (arr == ''){return null};
+let word = "";
+for (let i = 0; i < arr.length; i++) {
+ if (word.length < arr[i].length) {
+   word = arr[i];
+ }
+}
+return word;
+}
+
 // Iteration #3: Calculate the sum
 
 const numbers = [6, 12, 1, 18, 13, 16, 2, 1, 8, 10];
 
+function sumNumbers(arr){
+let sum = 0;
+  for (let i = 0; i < arr.length; i++){
+      sum = sum + arr[i];
+  }
+  return sum;
+}
+
 // Iteration #4: Calculate the average
 // Level 1: Array of numbers
 const numbersAvg = [2, 6, 9, 10, 7, 4, 1, 9];
 
+function averageNumbers(arr){
+if (arr == ''){return null};
+let average = sumNumbers(arr)/ arr.length;
+return average;
+}
+
 // Level 2: Array of strings
 const wordsArr = ['seat', 'correspond', 'linen', 'motif', 'hole', 'smell', 'smart', 'chaos', 'fuel', 'palace'];
 
+function averageWordLength(arr){
+if (arr == ''){return null};
+let lenghtsOfArrays = [];
+for (let i = 0; i < arr.length; i++){
+ lenghtsOfArrays.push(arr[i].length);
+}
+return (sumNumbers(lenghtsOfArrays)/arr.length);
+}
+
 // Iteration #5: Unique arrays
 const wordsUnique = [
-  'crab',
-  'poison',
-  'contagious',
-  'simple',
-  'bring',
-  'sharp',
-  'playground',
-  'poison',
-  'communion',
-  'simple',
-  'bring'
+'crab',
+'poison',
+'contagious',
+'simple',
+'bring',
+'sharp',
+'playground',
+'poison',
+'communion',
+'simple',
+'bring'
 ];
 
+function uniquifyArray(arr){
+if (arr == ''){return null};
+let newArray = [];
+
+for(let i = 0; i < arr.length; i++){
+ if(newArray.indexOf(arr[i]) === -1){
+     newArray.push(arr[i]);
+ } 
+} 
+return newArray;
+}
+
+
 // Iteration #6: Find elements
 const wordsFind = ['machine', 'subset', 'trouble', 'starting', 'matter', 'eating', 'truth', 'disobedience'];
 
+function doesWordExist(arr, word) {
+if (arr == ''){return null};
+if (arr.includes(word)){return true} else {return false}
+}
```

Comment:
in this case, you can also do
```return arrayOfWords.includes(word);```
because .includes() method return true or false ;)

Reply:
thanks!! 🙏